src/main/java/org/sagebionetworks/bridge/config/PropertiesConfig.java

```diff
@@ -0,0 +1,243 @@
+
+    static final String USER_KEY = "bridge.user";
+    static final String ENV_KEY = "bridge.env";
+
+    private final String user;
+    private final Environment environment;
+    private final Properties properties;
+    private final Pattern delimiter;
+
+    /**
+     * @param configTemplate
+     *            Path to the config template file in the source code. The
+     *            config template should contain all the config entries. The
+     *            values can be default values, or, if sensitive, dummy values.
+     *            The values on the template file are to be overwritten by
+     *            environment variables and/or system properties in that order.
+     */
+    public PropertiesConfig(final Path configTemplate) throws IOException {
+        this(configTemplate, configTemplate, DEFAULT_LIST_DELIMITER);
+    }
+
+    /**
+     * @param configTemplate
+     *            Path to the config template file in the source code. The
+     *            config template should contain all the config entries. The
+     *            values can be default values, or, if sensitive, dummy values.
+     *            The values on the template file are to be overwritten by those
+     *            of the local config file, environment variables, and/or system
+     *            properties in that order.
+     *
+     * @param userConfig
+     *            Path to the default local config file which is inside the
+     *            user's home directory.
+     */
+    public PropertiesConfig(final Path configTemplate, final Path userConfig) throws IOException {
+        this(configTemplate, userConfig, DEFAULT_LIST_DELIMITER);
+    }
+
+    /**
+     * @param configTemplate
+     *            Path to the config template file in the source code. The
+     *            config template should contain all the config entries. The
+     *            values can be default values, or, if sensitive, dummy values.
+     *            The values on the template file are to be overwritten by those
+     *            of the local config file, environment variables, and/or system
+     *            properties in that order.
+     *
+     * @param userConfig
+     *            Path to the default local config file which is inside the
+     *            user's home directory.
+     *
+     * @param delimiterRegex
+     *            The regular expression for the delimiter that is used to
+     *            separate a list of values.
+     */
+    public PropertiesConfig(final Path configTemplate, final Path localConfig,
+            final String delimiterRegex) throws IOException {
+        checkNotNull(configTemplate);
+        checkNotNull(localConfig);
+        checkNotNull(delimiterRegex);
+        final Properties properties = new Properties();
+        try (final Reader templateReader = Files.newBufferedReader(configTemplate);
+                final Reader localReader = Files.newBufferedReader(localConfig)) {
```

Comment:
I am aware of this.  The constructor takes a Path parameter.  It can be any file path.  It's up to the client to provide the path.  One way is to load it from the classpath like you are showing here.  That said, documentation is incorrect.  I have updated the documentation.